pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -326,6 +332,9 @@ static __always_inline void process_headers(struct __sk_buff *skb, dynamic_table
             read_into_buffer_path(dynamic_value.buffer, skb, current_header->new_dynamic_value_offset);
             bpf_map_update_elem(&http2_dynamic_table, dynamic_index, &dynamic_value, BPF_ANY);
             current_stream->path_size = current_header->new_dynamic_value_size;
+            // As long as we are not using the dynamic table, we can assume that the string is huffman encoded.
+            // if the string is indexed it would still be huffman encoded.
```

Comment:
not sure I'm following the documentation here

Reply:
redundant i have removed it